snyk-sdk/src/main/java/io/snyk/sdk/Snyk.java

```diff
@@ -51,23 +73,31 @@ public SnykClient buildSync() {
     String baseUrl;
     String token;
     String userAgent;
+    boolean trustAllCertificates;
+    String sslCertificatePath;
 
     public Config(String token) {
-      this.baseUrl = DEFAULT_BASE_URL;
-      this.token = token;
-      this.userAgent = DEFAULT_USER_AGENT;
+      this(DEFAULT_BASE_URL, token);
```

Comment:
@yuliabaron, no. It's chain constructor `this(DEFAULT_BASE_URL, token)` will call line 84 and then line 88 etc.
p.s.: we'll change this later to builder pattern, so it will not so confusing.